Code/Editor/Platform/Mac/main_dummy.cpp

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Similar to the concerns I noted for the corresponding cmake change above. The only difference between this and the AP version of this is that the executable we end up calling has a different name. Is there a better place for this where we can put the common code and just swap out the executable name? Again, I felt this isn't something that would be needed in most cases so I ended up creating 2 separate files which are mostly identical.

Reply:
yes, if you look at what we did with the launcher, I would suggest something similar for this.
Basically, this can be in a target that some `ly` function in cmake instantiates by just passing some defines. The binary to call and any other details that are process specific can be defines that cmake passes. CMake can create different targets with the same code with different defines which will give you the different wrappers.
We could create the wrappers automatically for any target of type "APPLICATION" (since those are the ones that will create bundles). We could even output the same binary name so you call the "Editor" executable and that launches the `Editor.app` bundle under the hood